{
  checkThread();
  try {
    Assert.assertSame(message,expected,actual);
  }
 catch (  AssertionError e) {
    handleThrowable(e);
  }
}
